### PR TITLE
Adding Estuary to umbrella

### PR DESCRIPTION
**Pull request by sukaant**

### CODE REVIEW

File: apps/estuary/config/prod.exs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+use Mix.Config
+
+# Do not print debug messages in production
```
Comment (ManApart) [inline apps/estuary/config/prod.exs]: Are these comments helpful? (this and the one in the test config)

File: apps/estuary/test/unit/estuary_test.exs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+defmodule EstuaryTest do
```
Comment (ManApart) [inline apps/estuary/test/unit/estuary_test.exs]: I think we can safely delete this module/file at this point.

Comment (dunson062786) [reply apps/estuary/test/unit/estuary_test.exs]: fixed

File: apps/estuary/config/prod.exs (hunk 1 above)
Comment (dunson062786) [reply apps/estuary/config/prod.exs]: Sorry, I thought I had removed this earlier. It's gone now.

**Pull request merged**